docs/cloud/azureai/flow-example-sharing-in-organization.md

```diff
@@ -0,0 +1,81 @@
+
+```yaml
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: <model_name>
+type: custom_model
+path: <the relative path from this model file to the flow folder>
+description: <model_description>
+version: <model_version>
+properties:
+  is-promptflow: true
+  azureml.promptflow.section: gallery
+  azureml.promptflow.type: <flow_type>
+  azureml.promptflow.name: <flow_name>
+  azureml.promptflow.description: <flow_description>
+```
+
+Properties released to UI display of this flow:
+1. `is-promptflow`: value should always be `true`. This property distinguishes it from other models, enabling promptflow service to filter it out.
+2. `azureml.promptflow.section`: value should always be `gallery`. This property indicates UI that this flow needs to be shown in the Flow Gallery.
+3. `azureml.promptflow.type`: value can be `chat`, `standard` or `evaluate`. This property identifies the type of your flow, and the UI will display different types of flows under different tabs accordingly.
+4. `azureml.promptflow.name`: the name of the flow which will be shown as the flow name in the Flow Gallery.
+5. `azureml.promptflow.description`: the description of the flow which will be shown as flow description in the Flow Gallery.
+
+Take [this existing flow](https://github.com/microsoft/promptflow/tree/main/examples/flows/chat/chat-with-wikipedia) as an example, the `model.yml` for it will look like below:
+```yaml
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: chat-with-wikipedia-in-org
+type: custom_model
+path: chat_with_wikipedia
+description: promptflow flow registered as a custom model
+version: 1
+properties:
+  is-promptflow: true
+  azureml.promptflow.section: gallery
+  azureml.promptflow.type: chat
+  azureml.promptflow.name: Chat with Wikipedia in Org
+  azureml.promptflow.description: ChatGPT-based chatbot that leverages Wikipedia data to ground the responses.
+```
+
+### Register the model in an organization registry
+
+ Run the command below to register the flow as a model to an organization registry:
+```
+az ml model create -f model.yml --registry-name <organization-registry-name>
+```
+
+> [!Note] More details about [registry creation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-manage-registries?view=azureml-api-2&tabs=studio#create-a-registry) and [model creation](https://learn.microsoft.com/en-us/cli/azure/ml/model?view=azure-cli-latest#az-ml-model-create)
+
+## Locate the flow in the Flow Gallery
+
+- Open a workspace in a region supported by the organization registry.
+- Click the `Create` button to open the Flow Gallery, and you can find the flow registered before:
+
+![organization examples in flow gallery](../../media/cloud/azureml/org_examples_in_flow_gallery.png)
```

Comment:
Can customer identity whether the flow is from system registry or org registry?

Reply:
Customer cannot identity whether the flow is from system registry or org registry since there is no any UI indicator for this.